packages/components/src/theme/ComposerTheme/Components/Radio.ts

```diff
@@ -0,0 +1,75 @@
+import { radioAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import { PartsStyleObject } from '@chakra-ui/theme-tools';
+
+const sizes: Record<string, PartsStyleObject<typeof parts>> = {
+  sm: {
+    label: { textStyle: 'text-body-meta-regular' },
+    control: {
+      _checked: {
+        border: '4px solid var(--chakra-colors-primary-default)',
+        _hover: {
+          border: '4px solid var(--chakra-colors-primary-hover)',
+        },
+      },
+    },
+  },
+  md: {
+    label: { textStyle: 'text-body-regular' },
+    control: {
+      _checked: {
+        border: '5px solid var(--chakra-colors-primary-default)',
+        _hover: {
+          border: '5px solid var(--chakra-colors-primary-hover)',
+        },
+      },
+    },
+  },
+  lg: {
+    label: { textStyle: 'text-body-large-regular' },
+    control: {
+      _checked: {
+        border: '6px solid var(--chakra-colors-primary-default)',
```

Comment:
Can't we use borderColor with the color token instead of using chakra css vars?

Reply:
I reorganized the component I was using the sizes when It was not necessary